FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,51 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let self = self else { return }
+
+			switch result {
+			case let .success((data, response)):
+				completion(FeedImagesMapper.map(data, from: response))
+
+			default:
+				completion(self.failure(.connectivity))
```

Comment:
Private helper here only helps to omit `Error` type/namespace, on the other hand it adds 3 more lines of code and makes the developer to use `self`. Code would be much simpler if the helper is inlined.

Please inline the helper:

```suggestion
				completion(.failure(Error.connectivity))
```